test/controllers/WebTest.java

```diff
@@ -147,7 +147,25 @@ private void checkSparkJobExceptionDetailElementForFlow2(JsonElement jobExceptio
       Assert.assertEquals(applicationJsonObject.get(JsonKeys.NAME).getAsString(), "application_id_1");
       JsonArray taskExceptionDetail = applicationJsonObject.getAsJsonArray(JsonKeys.TASKS);
       Assert.assertEquals(taskExceptionDetail.size(), 0);
-      Assert.assertEquals(applicationJsonObject.get(JsonKeys.EXCEPTION_SUMMARY).getAsString(), "stack_trace_3");
+      Assert.assertEquals(applicationJsonObject.get(JsonKeys.EXCEPTION_SUMMARY)
```

Comment:
nit: Creating `exceptionSummaryElementJsonObject` would be more efficient and would increase readability.
`exceptionSummaryElementJsonObject = applicationJsonObject.get(JsonKeys.EXCEPTION_SUMMARY)
          .getAsJsonArray()
          .get(0)
          .getAsJsonObject()`